src/video_core/renderer_opengl/renderer_opengl.h

```diff
@@ -59,6 +60,7 @@ class RendererOpenGL : public RendererBase {
     void DrawScreens();
     void DrawSingleScreenRotated(const ScreenInfo& screen_info, float x, float y, float w, float h);
     void UpdateFramerate();
+    void FrameLimiter(Common::Profiling::Duration average_frame_time, int frame_limit);
```

Comment:
It feels like this should use some more standard format for the frame time.
Including the profiling code to pass a real number [float] or integer [us] seems a bit overkill.